tests/util/kartutil.go

```diff
@@ -9,6 +9,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	types2 "github.com/onsi/gomega/types"
```

Comment:
```suggestion
	gomegaTypes "github.com/onsi/gomega/types"
```
I don't think naming it types2 would be a good practice